updating synthetics availability phrasing (#4353)
DataDog/documentation: content/en/synthetics/_index.md

  text: "Identify Synthetics Bots"
---

> <div class="alert alert-warning">Synthetics is only available on our US hosted instances. Browser tests are available in beta: to request access complete the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics Request form</a>.</div>

Reviewer: ```suggestion
<div class="alert alert-warning">Synthetics is only available for Datadog US site. Browser tests are available in beta: to request access complete the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics Request form</a>.</div>
```
